src/proto/grpc/binary_log/v1alpha/log.proto

```diff
@@ -105,4 +105,4 @@ message Message {
   // The contents of the message. May be a prefix instead of the complete
   // message.
   bytes data = 5;
-}
```

Comment:
: |
I'm not even sure how to remove this.  The git cli won't register a diff like this, so even after 
```git checkout origin/master log.proto``` I'm willing to bet this still shows up in the github diff.